src/components/organisms/AdminVenueView/components/DateTimeField/DateTimeField.scss

```diff
@@ -0,0 +1,22 @@
+@import "scss/constants.scss";
+
+.DateTimeField {
+  &__title {
+    margin: $spacing--lg 0 0;
+  }
+
+  &__subtitle {
+    font-weight: $font-weight--300;
+    color: opaque-white(0.6);
+    margin: 0;
+  }
+
+  &__container {
+    display: flex;
+    margin-top: $spacing--md;
```

Comment:
For some reason there are different margins that make the container go out of the left sidebar. I managed to fix it with 

```scss
  padding: 1em;
```

**Change**:
![sparkle-padding-03](https://user-images.githubusercontent.com/79229621/132858118-10e8bb6f-32df-4124-ad40-b5c8537f71db.png)

**Before**:
![sparkle-padding-02](https://user-images.githubusercontent.com/79229621/132858121-94159f66-225b-4630-9263-159367e8b503.png)

**After**:
![sparkle-padding-01](https://user-images.githubusercontent.com/79229621/132858126-3c9dbd23-c28b-4a6e-ba46-d000294b219b.png)

Reply:
fixed in ecdf3dcd597340a1deea0d0f04f0bc2037c2dc65

Reply:
LOL, wrong place for the above comment. Fixed in `Timing.scss`